data/footer.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 logo: "images/chips_alliance.svg"
 logo_url: ""
+Second Logo: "images/lf-stacked-white.svg"
```

Comment:
A few nits here:

1. I would name this something that is more semantic, since second logo does not mean much in this file and if the designed changed, it would not be second. Something like lf_logo
2. Related to the above, you should follow the same format as the rest of the items in the .yaml file, so all lowercase and use underscore instead of spaces.